### ISSUE TITLE
Midterm Peer Review

### ISSUE THREAD

**Issue by eweed001** [amg369/ORIE_4741_Project#5]
This project aims to predict the mood of a song based on the lyrics.

Good job gathering and preprocessing the data. One question I had here is how did you make sure you scraped the correct song? There’s a lot of songs with the same title. Did the ‘Moody Lyrics’ data set also have a column with artist name or was it just the title of the song and the labeled mood? I am wondering because if you have lyrics for a song with the same title as another but it resonates a completely different mood this would cause a big problem in the accuracy of your model. (‘One’ by Ed Sheehan vs. ‘One’ by Metallica).

I like the approach of using Naïve Bayes classification. It seems interpretable and good for the amount of data you have. Also using cross validation is definitely helpful. It might be good to specify how many folds you are using, instead of just saying ‘k-fold’. 

I like the inclusion of the correlation matrix. One question I had here was how did you make the determination for what to consider as ‘stopwords’? I saw that ‘I’ was considered a stop word, but ‘I’m’ is included in the correlation matrix. Also it is interesting to note that there were no words that were negatively correlated with each other (your scale only went 0-1 not -1-1). 

I think looking at a neural net would be very useful since you were overfitting so much with this approach. It sounds like you have a lot of steps lined up for the next portion of the project which is good. 

Another thing I might investigate if you haven’t already is the proportion of each mood label in your data set. You may have already taken this into account earlier but if there are few songs with the mood ‘relaxed’ for example it could skew your results. This could help when determining your loss function. 
Overall good job!
